###MY_VER_RANGE [4.0_alpha,5.0_alpha) [5.0_alpha,5.0.5_beta)

# for correct hardcoded sysconf directory

--- mysql-4.1.8/mysys/default.c	2004-12-14 13:40:36.000000000 +0100
+++ bbb/mysys/default.c	2005-01-09 17:57:00.407231408 +0100
@@ -48,7 +48,7 @@
 #elif defined(__NETWARE__)
 "sys:/etc/",
 #else
-"/etc/",
+"/etc/mysql/",
 #endif
 #ifdef DATADIR
 DATADIR,
